neuralop/training/incremental.py

```diff
@@ -0,0 +1,161 @@
+""" 
+Trainer for Incremental-FNO
+"""
+import torch
+from torch import nn
+
+from .trainer import Trainer
+from ..models import FNO, TFNO
+from ..utils import compute_explained_variance
+
+class IncrementalFNOTrainer(Trainer):
+    """IncrementalFNOTrainer subclasses the Trainer 
+    to implement specific logic for the Incremental-FNO
+    as described in [1]
+
+    References
+    -----------
+    [1] George, R., Zhao, J., Kossaifi, J., Li, Z., and Anandkumar, A. (2024)
```

Comment:
It still isn't a correct Rest reference, the text above in the description should be `[1]_` to link to this reference. Please check the doc https://docutils.sourceforge.io/docs/user/rst/quickref.html